### DISCUSSION TITLE
Alignment in GroupBox

### DISCUSSION THREAD

**Discussion by Futsch1** [slint-ui/slint#949] [Q&A] [answered]
It seems that the alignment property in a GroupBox does not work on all systems. I have received a bug report (see https://github.com/Futsch1/image-sieve/issues/30), but cannot reproduce the behavior, neither can Github Actions (Ubuntu + Windows).

What I wasn't able to find out: Is the alignment property valid in a GroupBox?

**Comment by ogoffart** [comment, marked as answer]
Short answer: no: it's not.
This happens to work in the `fluent` style because we are leaking some implementation details and that the fluent style uses a layout behind the scene. But with the `native` style (which is the default if Qt is installed) this property does not exist.
That property is not documented and only exported by accident.

Maybe we should document and implement this property on all styles.

In the mean time, you should either not use this property (It is easy to simulate with an empty Rectangle {}  that does the spacing.)
Or you can force the "fluent" style with https://docs.rs/slint-build/latest/slint_build/struct.CompilerConfiguration.html#method.with_style in your build script.

**Comment by ogoffart** [comment]
I forgot to mention that if you're not wanting to use the native style, you may also want disable the Qt backend in the cargo feature flags of the slint crate
